src/Response/SapiEmitter.php

```diff
@@ -23,19 +23,13 @@ class SapiEmitter implements EmitterInterface
      * body content via the output buffer.
      *
      * @param ResponseInterface $response
-     * @param null|int $maxBufferLevel Maximum output buffering level to unwrap.
      */
-    public function emit(ResponseInterface $response, $maxBufferLevel = null)
+    public function emit(ResponseInterface $response)
     {
-        if (headers_sent()) {
-            throw new RuntimeException('Unable to emit response; headers already sent');
```

Comment:
Why was this removed? You won't be able to emit the status line or headers if output has already been started - the first call to `header()` will error. Isn't it better to have the exception, which explains why?

Reply:
Oh, never mind, you just moved it somewhere else ;-)

Reply:
:smile: